### DISCUSSION TITLE
I can't connect to a device

### DISCUSSION THREAD

**Discussion by igormcsouza** [shellhub-io/shellhub#2893] [Q&A] [answered]
A have added shellhub agent to the device as explained, but when I try to login using shellhub, I got erros, 

![image](https://github.com/shellhub-io/shellhub/assets/38996012/1659dfa2-63fa-444e-82a6-434f36d50021)

![image](https://github.com/shellhub-io/shellhub/assets/38996012/59890e48-0bc0-49f1-b933-2412e540bbe2)

When I try to ssh direct to this device with same user and password, I can no problem! What should be my mistake?

**Comment by otavio** [comment]
What is your OS (in the device)?

Could you check `docker logs shellhub`?

**Reply by igormcsouza** [reply]
I'm using Ubuntu 20.04.6

```
~ ➜ docker logs shellhub
time="2023-06-15T17:56:00Z" level=info msg="Setting log level" log_level=info
time="2023-06-15T17:56:00Z" level=info msg="Starting ShellHub" mode=multi-user version=v0.12.2-rc.6
2023/06/15 17:56:20.663389 ERROR RESTY Get "http://cloud.shellhub.io:80/info?agent_version=v0.12.2-rc.6": dial tcp: lookup cloud.shellhub.io on 127.0.0.53:53: read udp 127.0.0.1:33145->127.0.0.53:53: i/o timeout, Attempt 1
time="2023-06-15T18:11:32Z" level=info msg="Setting log level" log_level=info
time="2023-06-15T18:11:32Z" level=info msg="Starting ShellHub" mode=multi-user version=v0.12.2-rc.6
time="2023-06-15T18:16:16Z" level=info msg="Setting log level" log_level=info
time="2023-06-15T18:16:16Z" level=info msg="Starting ShellHub" mode=multi-user version=v0.12.2-rc.6
```

**Comment by otavio** [comment, marked as answer]
We have implemented a temporary solution and are currently working on properly resolving the issue. Kindly execute the following commands:
```
docker stop shellhub
docker rm shellhub
```
and reinstall the agent. It should work.

**Comment by igormcsouza** [comment]
Thanks you! It worked! :D I'll be continue looking at it on the ISSUE side! :) I appreciate!

**Reply by otavio** [reply]
See: #2894 👍🏼
